lib/matplotlib/text.py

```diff
@@ -2181,12 +2184,13 @@ def transform(renderer) -> Transform
         else:
             self.arrow_patch = None
 
-    def contains(self, event):
-        contains, tinfo = Text.contains(self, event)
+    def contains(self, mouseevent):
```

Comment:
The change of the parameter name is an API change. While, naming of this parameter is not consistent throughout the library, I prefer not to sneek this in with the PR.

That should be handled in a separate PR and unify all `contains` signatures. It needs at least an API-change note (if  not a `@cbook._rename_parameter` deprecation).

Reply:
reverted that change